src/categorical_algebra/FinSets.jl

```diff
@@ -454,9 +460,147 @@ struct FinSetFreeDiagramLimit{Ob<:FinSet, Diagram<:FreeDiagram{Ob},
   incl::Incl # Inclusion for the "multi-equalizer" in general formula.
 end
 
+function limit(d::BipartiteFreeDiagram{Ob,Hom}) where
```

Comment:
OK, this is the function that I'm going to use downstream for ACSet limits, correct?

Reply:
I hope so! Is this class of limits general enough to do products and pullbacks of ACSets? I have not thought through how that would work.